frontend/components/ArticleBody.tsx

```diff
@@ -475,44 +475,47 @@ const bylineAsTokens = (bylineText: string, tags: TagType[]): string[] => {
     return bylineText.split(regex);
 };
 
-const renderByline = (
-    bylineText: string,
-    contributorTags: TagType[],
-    pillar: Pillar,
-): JSX.Element[] => {
+const RenderByline: React.SFC<{
+    bylineText: string;
+    contributorTags: TagType[];
+    pillar: Pillar;
+}> = ({ bylineText, contributorTags, pillar }) => {
     const renderedTokens = bylineAsTokens(bylineText, contributorTags).map(
-        token => {
+        (token, i) => {
             const associatedTags = contributorTags.filter(
```

Comment:
This is a good example of some data modelling that could be pulled out of a Component's lifecycle and done once on the server (in frontend) as there's no need to do this on the server and then again in the client.